content/en/docs/reference/glossary/cluster.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Cluster
```

Comment:
I'm assuming you borrowed liberally from Kubernetes here. I don't think we should use their definition. We need to be more generic ... Istio is not k8s-specific, and neither is the concept of a cluster.

Reply:
I agree, this needs to be a more generic definition of cluster, not the K8s one.

Reply:
I'm replacing the definitions with the following platform agnostic version: 
```
A set of compute nodes within physical proximity that run containerized
applications.

Typically, the compute nodes comprising a cluster can reach each other directly.
```
WDYT?

Reply:
I think that's a lot better ... let's switch to this and let folks throw darts at it.